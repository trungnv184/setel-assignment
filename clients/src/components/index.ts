export { default as Header } from './Header/Header';
export { default as ProductItem } from './ProductItem/ProductItem';
export { default as ProductList } from './ProductList/ProductList';
export { default as Layout } from './Layout/Layout';
export { default as CartList } from './Cart/CartList';
export { default as MenuBar } from './MenuBar/MenuBar';
export { default as ErrorMessage } from './ErrorMessage/ErrorMessage';
export { default as OrderForm } from './OrderForm/OrderForm';
export { StyledAppContainer } from './Common/Style';
