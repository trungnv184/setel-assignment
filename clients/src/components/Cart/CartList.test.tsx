import { render } from '@testing-library/react';
import { CartList } from 'components';
import { CARTS_MOCK } from 'test/mocks/carts.mock';

describe('CartList', () => {
  test('should match snapshot', () => {
    const { container } = render(<CartList carts={CARTS_MOCK} />);
    expect(container).toMatchSnapshot();
  });
});
