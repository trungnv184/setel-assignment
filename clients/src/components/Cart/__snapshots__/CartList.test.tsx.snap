// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartList should match snapshot 1`] = `
<div>
  <table
    class="ui celled padded table"
  >
    <thead
      class=""
    >
      <tr
        class=""
      >
        <th
          class="single line"
        >
          Image
        </th>
        <th
          class=""
        >
          Name
        </th>
        <th
          class=""
        >
          Price
        </th>
        <th
          class=""
        >
          Quantity
        </th>
      </tr>
    </thead>
    <tbody
      class=""
    >
      <tr
        class=""
      >
        <td
          class=""
        >
          <img
            class="ui small image"
            src="https://salt.tikicdn.com/cache/280x280/ts/product/59/e3/e7/642ce38017ac1d35f2e87234b6fd56f2.jpg"
          />
        </td>
        <td
          class="single line"
        >
          Tâm lý học hạnh phúc
        </td>
        <td
          class=""
        >
          200.000 ₫
        </td>
        <td
          class=""
        >
          1
        </td>
      </tr>
      <tr
        class=""
      >
        <td
          class=""
        >
          <img
            class="ui small image"
            src="https://salt.tikicdn.com/cache/280x280/ts/product/5f/36/dd/c64c91016b320ae4591efcf93d0ebf30.jpg"
          />
        </td>
        <td
          class="single line"
        >
          Dạy con học ít hiểu nhiều
        </td>
        <td
          class=""
        >
          200.000 ₫
        </td>
        <td
          class=""
        >
          1
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
